optuna/pruners/asha.py

```diff
@@ -0,0 +1,145 @@
+import math
+
+from optuna.pruners.base import BasePruner
+from optuna.storages import BaseStorage  # NOQA
+from optuna.structs import FrozenTrial  # NOQA
+from typing import List  # NOQA
+
+
+class ASHAPruner(BasePruner):
```

Comment:
Do you prefer `ASHAPruner` over other descriptive names such as `SuccessiveHalvingPruner`? I think the name `ASHA` is not very popular yet.

Reply:
Done: https://github.com/pfnet/optuna/pull/236/commits/e184402baaf4ad41cad6983052032188e13f583b